WordPress/Classes/Stores/RemoteFeatureFlagStore.swift

```diff
@@ -66,6 +68,8 @@ extension RemoteFeatureFlagStore {
     struct Constants {
         static let DeviceIdKey = "FeatureFlagDeviceId"
         static let CachedFlagsKey = "FeatureFlagStoreCache"
+        static let LastRefreshDateKey = "FeatureFlagLastRefreshDate"
+        static let CacheTTL: TimeInterval = 86_400 // 24 hours
```

Comment:
Adding this TTL makes this feature considerably less useful – consider a situation in which the app were overloading our API due to a feature flag change. Being forced to wait 24 hours would be very unfortunate.

Do we need this?

Reply:
With the original design, we used to only refresh the feature flags on launch. Meaning we could have to wait days or weeks or more in some cases. We changed this to refresh the flags on every launch and relaunch. And we added this TTL to avoid unnecessarily spamming the endpoint. These two changes ensure we frequently update the flags, but not too frequently.

Ref: p1661916980513999-slack-C03T04HETR8

Reply:
One thing I missed is that we agreed on a 12-hour TTL, not 24 hours. Updated here: 2487d0de591c8285cbc70cc69cc1341d8dc664cd

Reply:
I'm surprised this cache wouldn't be implemented server-side though – that way we'd have the ability to clear it and pay the extra server cost in a situation where we needed to change something quickly. Any TTL (no matter how short) will feel likes _ages_ in an emergency 😅

WDYT?

Reply:
> I'm surprised this cache wouldn't be implemented server-side

My understanding is that there's a cache level on the server-side as well. The app level TTL is intended to avoid overloading the new endpoint. But you're posing a good point though.

@momo-ozawa @zwarm @AjeshRPai What do you think?

Reply:
I remove the cache logic in c7f108619580e131fef3855c48290ccaedd2d4ee